src/popup/components/Card.vue

```diff
@@ -71,12 +71,19 @@ export default {
     disabled: Boolean,
     cardId: { type: String, default: null },
     icon: { type: Object, default: null },
+    bgDarken: { type: Boolean, default: false },
+    bgPosition: {
+      type: String,
+      default: null,
+    },
   },
   setup(props) {
     const hiddenCards = useState('hiddenCards');
 
+    const bgImage = computed(() => props.bgDarken ? `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("${props.background}")` : `url("${props.background}")`);
```

Comment:
I suggest to avoid putting CSS rules in Javascript block. I suggest to add/remove class `bg-darken` based on the `bgDarken` prop and allow this class to take care of the styling.

Next thing: why `linear-gradient`? Maybe `::before` pseudo class with opacity would be better?

Reply:
I am not sure, opacity will affect also the text inside of element. We only want to darken the background

Reply:
If you apply the opacity to pseudo element (like `::before` or `::after`) it won't affect other layers.